src/main/java/calculator/PositiveNumbers.java

```diff
@@ -0,0 +1,53 @@
+package calculator;
+
+public class PositiveNumbers {
+
+    private final int[] numbers;
+
+    public PositiveNumbers(int[] numbers) {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+    public PositiveNumbers(String[] values) {
+        this(ofStringValues(values));
+    }
+
+    public int sum() {
+        int sum = 0;
+        for (int number : numbers) {
+            sum += number;
+        }
+        return sum;
```

Comment:
스트림을 활용해서 이렇게도 쓸 수 있을거 같아요. 
```suggestion
        return Arrays.stream(numbers).sum();
```